src/Services/PopoverManager.vala

```diff
@@ -60,16 +60,9 @@ public class Wingpanel.Services.PopoverManager : Object {
                 popover.popup ();
                 popover.show_all ();
                 _current_indicator.base_indicator.opened ();
-                is_popover_open = true;
-                _current_indicator.display_widget.notify["has-tooltip"].connect (() => {
-                    if (is_popover_open) {
-                        _current_indicator.display_widget.has_tooltip = false;
-                    }
-                });
             } else {
-                ((Wingpanel.Widgets.IndicatorEntry)popover.get_relative_to ()).indicator_widget.has_tooltip = true;
+                ((Wingpanel.Widgets.IndicatorEntry)popover.get_relative_to ()).display_widget.has_tooltip = true;
```

Comment:
@jeremypw this is where I replaced the `.indicator_widget` with `.display_widget` FYI to take a look if it's OK or not?

Reply:
You are right, it should be display_widget as that is what the tooltip is attached to.  I must got confused with the naming.